project/Game.py

```diff
@@ -170,6 +174,24 @@ def modify_landmark(self, oldclue, clue=None, question=None, answer=None):
         except KeyError:
             return False
 
+    def edit_landmark_order(self, oldindex, newindex):
+        if not self.started:
+            try:
+                index1 = int(oldindex)
+                index2 = int(newindex)
+            except ValueError:
+                return Errors.STRING_TO_INTEGER
+            if index1 < 0 or index2 < 0:
```

Comment:
Do we need to check specifically for a negative index? Can't we just try and use it and get the IndexError?

Reply:
hah, look at that, negative values start from the end of the list... Do we really need to restrict that weird, but neat, Python quirk?

Reply:
lol um so negative values are okay?

Reply:
In python it seems so.
```>>> mine = [1,2,3,4]
>>> mine[-1]
4
>>> mine[-2]
3
>>> mine[-4]
1
>>> mine[-5]```

Reply:
After a few edits my example includes -5. That's a crash

Reply:
-4 to 3 are valid indexes, everything else is an index error

Reply:
hmm so we could just limit it values > -n, break out of method

Reply:
alright